openfisca_france/model/prelevements_obligatoires/impot_revenu/ir.py

```diff
@@ -3496,7 +3496,7 @@ def formula(foyer_fiscal, period, parameters):
         abattements = parameters(period).impot_revenu.calcul_revenus_imposables.abat_rni
 
         # Abattement pour personnes agées de + de 65 ans ou invalide
-        abattement_age_ou_invalidite = abattements.personne_agee_ou_invalide
+        abattement_age_ou_invalidite = abattements.contribuable_agee_invalide
```

Comment:
```suggestion
        abattement_age_ou_invalidite = abattements.contribuable_age_invalide
```
A remplacer partout je pense.

Reply:
Merci Sylvain j'avais aussi repéré la typo et j'ai oublié de faire le changement, je m'en occupe ainsi que pour la doc